src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/PanelMemberSpecialismsMapping.java

```diff
@@ -0,0 +1,59 @@
+package uk.gov.hmcts.reform.sscs.helper.mapping;
+
+import uk.gov.hmcts.reform.sscs.ccd.domain.PanelMember;
+import uk.gov.hmcts.reform.sscs.ccd.domain.PanelMemberMedicallyQualified;
+import uk.gov.hmcts.reform.sscs.ccd.domain.SscsCaseData;
+import uk.gov.hmcts.reform.sscs.reference.data.model.SessionCategoryMap;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import javax.validation.Valid;
+
+import static java.util.Objects.isNull;
+import static java.util.Objects.nonNull;
+import static uk.gov.hmcts.reform.sscs.ccd.domain.PanelMemberMedicallyQualified.getPanelMemberMedicallyQualified;
+
+public final class PanelMemberSpecialismsMapping {
+
+    private PanelMemberSpecialismsMapping() {
+
+    }
+
+    public static List<String> getPanelSpecialisms(@Valid SscsCaseData caseData, SessionCategoryMap sessionCategoryMap) {
+        List<String> panelSpecialisms = new ArrayList<>();
+
+        if (isNull(sessionCategoryMap)) {
+            return panelSpecialisms;
+        }
+
+        String doctorSpecialism = caseData.getSscsIndustrialInjuriesData().getPanelDoctorSpecialism();
+        String doctorSpecialismSecond = caseData.getSscsIndustrialInjuriesData().getSecondPanelDoctorSpecialism();
+        panelSpecialisms = sessionCategoryMap.getCategory().getPanelMembers().stream()
+            .map(panelMember -> getPanelMemberSpecialism(panelMember, doctorSpecialism, doctorSpecialismSecond))
+            .filter(Objects::nonNull)
+            .collect(Collectors.toList());
+        return panelSpecialisms;
+    }
+
+    public static String getPanelMemberSpecialism(PanelMember panelMember,
+                                                  String doctorSpecialism, String doctorSpecialismSecond) {
+        switch (panelMember) {
+            case MQPM1:
+                return getReference(doctorSpecialism);
+            case MQPM2:
+                return getReference(doctorSpecialismSecond);
+            default:
+                return panelMember.getReference();
+        }
+    }
+
+    public static String getReference(String panelMemberSubtypeCcdRef) {
+        PanelMemberMedicallyQualified subType = getPanelMemberMedicallyQualified(panelMemberSubtypeCcdRef);
+        return nonNull(subType)
+            ? subType.getHmcReference()
+            : null;
+    }
+
```

Comment:
Has this been removed from commons?

Reply:
Yeah has been remove in this PR https://github.com/hmcts/sscs-common/pull/1246